apps/web/src/index.scss

```diff
@@ -0,0 +1,54 @@
+body {
+  font-family: "Inter";
```

Comment:
do you fetch the font? I didn't find it. if not - an example is in desktop/index.html